From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: chiralpenguin <50334616+chiralpenguin@users.noreply.github.com>
Date: Fri, 10 Dec 2021 12:46:31 +0000
Subject: [PATCH] Fix imports

Remove imports from PurpurConfig and PurpurWorldConfig which were left over from the rebuild of the fork

diff --git a/src/main/java/net/pl3x/purpur/PurpurConfig.java b/src/main/java/net/pl3x/purpur/PurpurConfig.java
index 99801676052591439056920e0a03f690869dee39..9f1562f0ab476e058e1a4560a13794b5431b48d7 100644
--- a/src/main/java/net/pl3x/purpur/PurpurConfig.java
+++ b/src/main/java/net/pl3x/purpur/PurpurConfig.java
@@ -8,7 +8,6 @@ import net.minecraft.server.MinecraftServer;
 import net.minecraft.world.entity.EntityDimensions;
 import net.minecraft.world.entity.EntityType;
 import net.pl3x.purpur.command.PurpurCommand;
-import net.pl3x.purpur.task.TPSBarTask;
 import org.bukkit.Bukkit;
 import org.bukkit.command.Command;
 import org.bukkit.configuration.ConfigurationSection;
diff --git a/src/main/java/net/pl3x/purpur/PurpurWorldConfig.java b/src/main/java/net/pl3x/purpur/PurpurWorldConfig.java
index 49cad8f2c1e15a92dad4dc50e7c7743c490e8e69..9c05b2ba35e232a5b686b2c971e0d4c8c505d0b0 100644
--- a/src/main/java/net/pl3x/purpur/PurpurWorldConfig.java
+++ b/src/main/java/net/pl3x/purpur/PurpurWorldConfig.java
@@ -11,10 +11,6 @@ import net.minecraft.world.level.Explosion;
 import net.minecraft.world.level.block.Block;
 import net.minecraft.world.level.block.Blocks;
 import net.minecraft.world.level.block.state.properties.Tilt;
-import net.pl3x.purpur.tool.Strippable;
-import net.pl3x.purpur.tool.Tillable;
-import net.pl3x.purpur.tool.Waxable;
-import net.pl3x.purpur.tool.Weatherable;
 import org.apache.commons.lang.BooleanUtils;
 import org.bukkit.ChatColor;
 import org.bukkit.World;
